gate-web/src/main/groovy/com/netflix/spinnaker/gate/config/GateLoginImageConfigProperties.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2019 OpsMX, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.gate.config;
+
+import lombok.Data;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnExpression;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@ConditionalOnExpression("${security.login.image.enabled:false}")
```

Comment:
This should be a `ConditionalOnProperty`.

Reply:
Updated with ConditionalOnProperty.